rpc/src/commonMain/kotlin/foundation/metaplex/rpc/serializers/AnchorAccountSerializer.kt

```diff
@@ -0,0 +1,63 @@
+package foundation.metaplex.rpc.serializers
+
+import diglol.crypto.Hash
+import kotlinx.coroutines.runBlocking
+import kotlinx.serialization.KSerializer
+import kotlinx.serialization.descriptors.SerialDescriptor
+import kotlinx.serialization.encoding.Decoder
+import kotlinx.serialization.encoding.Encoder
+import kotlinx.serialization.serializer
+
+open class DiscriminatorSerializer<T>(val discriminator: ByteArray, serializer: KSerializer<T>)
```

Comment:
I'm not familiar with the concept of a "Discriminator". Is this a Solana thing? Can you add a description?

Reply:
I added more docs specially for anchor